spec/project/default_config_spec.rb

```diff
@@ -15,7 +15,8 @@
   end
 
   let(:cop_names) do
-    glob = SpecHelper::ROOT.join('lib', 'rubocop', 'cop', 'rspec',
+    glob = SpecHelper::ROOT.join('lib', 'rubocop', 'cop',
+                                 '{capybara,factory_bot,rspec,rspec-rails}',
                                  '{,capybara,factory_bot,rails}', '*.rb')
```

Comment:
Not perfect. Could probably be `'{capybara,factory_bot,rspec,rspec/capybara,rspec/factory_bot,rspec/rails,rspec-rails}`, but it's not platform-independent. Do we care about other platforms (that one with `\` file separator!) running our tests?